.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # Every request must be reviewed and accepted by:
 
-* @VeryGoodOpenSource/codeowners
+*	@alestiago @erickzanardo @felangel @marcossevilla @renancaraujo @wolfenrain
```

Comment:
Should we create a codeowners group in our org just like there were one at VGV? So we don't need to repeat ourselves?